### DISCUSSION TITLE
Devin's Projects has been rebranded to Skyward

### DISCUSSION THREAD

**Discussion by intergrav** [skywardmc/.github#1] [Announcements]
Devin's Projects has been rebranded to SkywardMC and repositories have been transferred to this GitHub organization. This has been done as I may soon have more people maintaining older projects of mine that I haven't been able to update or neglected, among other things. An organization makes this significantly easier to manage, and since it won't be just me, I wanted to move away from the "**Devin**'s Projects" name.

Consider joining the Discord server [here](https://discord.gg/36Tv44cYte) for more updates on SkywardMC projects!

**Comment by mochaaP** [comment]
Congrats! Is it possible to join the organization? I would like to help if possible :smiley:

**Comment by intergrav** [comment]
> Congrats! Is it possible to join the organization? I would like to help if possible 😃

What would you like to help out with? I can't think of any repos that need maintainers right now haha
